.vsts-ci.yml

```
@@ -93,7 +93,7 @@ stages:
     parameters:
       platform:
         name: 'Managed'
-        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8'
+        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0'
```

Source build legs should continue to use centos as we don't ship a SB product via mariner.

```suggestion
        container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9'
```